packages/embedded-wallet/src/ui/flow-process/FlowStep.test.tsx

```diff
@@ -0,0 +1,67 @@
+import Enzyme, {mount, ReactWrapper} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import {Icon, IconProps, Icons} from '../icon/Icon';
+import {Spinner} from '../spinner/Spinner';
+import {FlowStep, FlowStepProps, FlowStepStatus} from './FlowStep';
+import css from './FlowStep.module.css';
+
+Enzyme.configure({adapter: new Adapter()});
+
+type MockFlowStep = {
+  flowStepWrapper: ReactWrapper;
```

Comment:
If all elements are of the same type, except one, to improve the readability of the test, wouldn't it be better to use something like 
```
type MockFlowStep = {
    [key: string]: ReactWrapper,
    statusIconElement: ReactWrapper<IconProps>
}
```
Or something like that?

Reply:
Not really, I want the keys to be declared explicitly. `[key: string]` is great, but it won't help IntelliSense and makes the type more error-prone by allowing minor typos (i.e. `flowStepWrapper` vs `flowStepWraper` to happen).